src/components/AdviceGeneratorApp/AdviceGeneratorApp.tsx

```diff
@@ -1,15 +1,20 @@
-import { getAdvices } from "../../helpers/getAdvices"
+// import { getAdvices } from "../../helpers/getAdvices"
+import { useFetchAdvices } from "../../helpers/useFetchAdvices"
 
 export const AdviceGeneratorApp = () => {
 
-  getAdvices();
+  const { adviceState } = useFetchAdvices();
```

Comment:
Aquí es en donde utilizar el error y opcionalmente el loading

```suggestion
  const { advice, loading, error } = useFetchAdvices();
```